code/modules/urist/modules/resourcetech/tools.dm

```diff
@@ -33,4 +33,12 @@
 
 //huntergun
 
-//hunterknife
\ No newline at end of file
+//hunterknife
+
+/obj/item/weapon/material/knife/hunting
+	icon = 'icons/urist/items/uristweapons.dmi'
+	icon_state = "huntknife"
+	force = 10
```

Comment:
Don't quote me on this, but I *think* force won't do anything on material weapons, it'll use the material brute_mod.